.github/workflows/deploy_to_firebase_app_distribution.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy staging and production to Firebase App Distribution
+
+on:
+  # Trigger the workflow on push action in main branch.
+  # So it will trigger when the PR of the release branch was merged.
+  push:
+    branches:
+      - main
```

Comment:
this should be executed on `develop` 🤔

Reply:
@luongvo Nope 😄 I think we should follow the convention of doing merge release PR to `main` as normally. Or do you have any other idea on your suggestion?

Reply:
@hoangnguyen92dn you don't plan to release new build when merging PRs into develop? I think we can have sooner builds when switching to dev? 🤔 

My plan is:
- merging to dev > build staging
- merging to main > build production

Reply:
@luongvo Got your idea 💡I will update for the staging build once feature branches merged to `develop`

Reply:
@hoangnguyen92dn Would you mind creating a new task to distribute production app via main branch? 🤔